etc/meld.profile

```diff
@@ -10,7 +10,7 @@ noblacklist ${HOME}/.local/share/meld
 noblacklist ${PATH}/python*
 noblacklist /usr/include/python*
 noblacklist /usr/lib/python*
-noblacklist /usr/local/lib/python*
+noblacklist /usr/local/lib/python*,git,svn,hg,bzr,cvs
```

Comment:
find all and replace?

Reply:
not entered, but maybe a button accidentally pressed (using vim). will be removed.